src/_common/popper/popper.styl

```diff
@@ -7,92 +7,78 @@ $-transition-time = 200ms
 .v-popover
```

Comment:
Let's call this something other than v-popper.

Reply:
renamed to `popper`, and the wrapper for the actual popper/popup element is changed to `popper-wrapper`.